src/main/java/ee/tuleva/onboarding/payment/PaymentData.java

```diff
@@ -20,14 +20,16 @@ public class PaymentData {
   public enum PaymentType {
     SINGLE,
     RECURRING,
-    GIFT
+    GIFT,
+    MEMBER_FEE
   }
 
   public enum Bank {
     LUMINOR,
     SEB,
     SWEDBANK,
     LHV,
-    COOP
+    COOP,
+    TULUNDUSUHISTU
```

Comment:
Any other way to model this? 🤔 Tulundusühistu is not a bank.

Reply:
I think it makes sense now, we might want to rename the entity from Bank to PaymentChannel